
<pre>
MOD_PYTHON ERROR

ProcessId:      13383
Interpreter:    'reshuege.ru'

ServerName:     'reshuege.ru'
DocumentRoot:   '/var/www/reshuege/www'

URI:            '/index.py'
Location:       None
Directory:      '/var/www/reshuege/www/'
Filename:       '/var/www/reshuege/www/index.py'
PathInfo:       None

Phase:          'PythonHandler'
Handler:        'mod_python.publisher'

Traceback (most recent call last):

  File "/usr/lib/python2.7/dist-packages/mod_python/importer.py", line 1537, in HandlerDispatch
    default=default_handler, arg=req, silent=hlist.silent)

  File "/usr/lib/python2.7/dist-packages/mod_python/importer.py", line 1229, in _process_target
    result = _execute_target(config, req, object, arg)

  File "/usr/lib/python2.7/dist-packages/mod_python/importer.py", line 1128, in _execute_target
    result = object(arg)

  File "/usr/lib/python2.7/dist-packages/mod_python/publisher.py", line 213, in handler
    published = publish_object(req, object)

  File "/usr/lib/python2.7/dist-packages/mod_python/publisher.py", line 425, in publish_object
    return publish_object(req,util.apply_fs_data(object, req.form, req=req))

  File "/usr/lib/python2.7/dist-packages/mod_python/util.py", line 554, in apply_fs_data
    return object(**args)

  File "/var/www/reshuege/www/index.py", line 35, in index
    return t_index(req)

  File "/var/www/reshuege/www/index.py", line 493, in t_index
    import magic

ImportError: No module named magic


MODULE CACHE DETAILS

Accessed:       Mon May 13 14:20:37 2013
Generation:     1

_mp_509bc8ea1039ffba9e3c82d9e47bd50c {
  FileName:     '/var/www/reshuege/www/index.py'
  Instance:     1
  Generation:   1
  Modified:     Mon May 13 04:16:58 2013
  Imported:     Mon May 13 14:19:17 2013
}

</pre>
